terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars

```diff
@@ -0,0 +1,36 @@
+aws_bucket = "development-eu-west-2.terraform-state.ch.gov.uk"
+remote_state_bucket = "ch-development-terraform-state-london"
+environment = "cidev"
+deploy_to = "development"
+state_prefix = "env:/development"
+aws_profile = "development-eu-west-2"
+kms_alias = "alias/development-eu-west-2/environment-services-kms"
+
+# service configs
+# Commented params moved to vault service secrets
+# cache_server 
+# oauth2_redirect_uri 
+# oauth2_auth_uri 
+# account_url 
+log_level = "debug"
+chs_url = "https://cidev.aws.chdev.org"
+piwik_url = "https://matomo.platform.aws.chdev.org/"
+piwik_site_id = "7"
+cdn_host = "d3uvya5a8a1ncx.cloudfront.net"
+chips_presenter_auth_url = "http://enablement-presenter-api:8080/presenterauth"
```

Comment:
This looks like what could be a live value to be, does it need to make reference to cidev anywhere?

Reply:
(https://github.com/companieshouse/chs-configs/blob/develop/cidev/dissolution-web/env)
Value here is 
export CHIPS_PRESENTER_AUTH_URL=http://localhost:18211/presenterauth
I'll make that change